### ISSUE TITLE
Creating the AMI for CSHL 2022

### ISSUE THREAD

**Issue by chrisamiller** [griffithlab/rnabio.org#72]
Right now, the AMI seems to be missing a fair number of tools that will be needed for the course, mostly from pmbio lessons that will be used. Let's use this thread to organize those updates and make sure we're all on the same page.

- [x] Use checkboxes to indicate items to install, then they can be checked off as they get added

**Comment by chrisamiller** [comment]
Required software for the long read sequencing lesson:

- [x] Nanoplot: https://github.com/wdecoster/NanoPlot
- [x] Minimap2: https://github.com/lh3/minimap2/releases
- [ ] Pychopper: https://github.com/epi2me-labs/pychopper

**Comment by chrisamiller** [comment]
- [x] GATK isn't setup correctly. 
It seems to be installed - just running `gatk` does produce the help, but trying to run any command or list the available tools causes it to choke:

```
$ gatk --list
Using GATK jar /home/ubuntu/bin/gatk-4.4.0.0/gatk-package-4.4.0.0-local.jar
Running:
    java -Dsamjdk.use_async_io_read_samtools=false -Dsamjdk.use_async_io_write_samtools=true -Dsamjdk.use_async_io_write_tribble=false -Dsamjdk.compression_level=2 -jar /home/ubuntu/bin/gatk-4.4.0.0/gatk-package-4.4.0.0-local.jar --help
Error: LinkageError occurred while loading main class org.broadinstitute.hellbender.Main
	java.lang.UnsupportedClassVersionError: org/broadinstitute/hellbender/Main has been compiled by a more recent version of the Java Runtime (class file version 61.0), this version of the Java Runtime only recognizes class file versions up to 55.0
```

**Comment by chrisamiller** [comment]
For the somatic variant calling section: 

- [x] Mutect (should come for free with already-installed gatk, but verify that it works)
- [x] Varscan https://github.com/dkoboldt/varscan/releases/download/2.4.2/VarScan.v2.4.2.jar

We also use a tool (CombineVariants) from an older version of GATK (3.6).  I assume that dueling GATK installations might be a bad idea, so I think we just use docker for that one. It's in `mgibio/gatk-cwl:3.6.0` and is linked from the exercise

**Comment by mhoang22** [comment]
GATK: resolved. (problem: previously I installed gatk-4.2.1.0 , which used an more updated java version (java 17 I think). I resolved this by installing gatk-4.2.1.0 , which uses java 11 ) (https://stackoverflow.com/questions/47457105/class-has-been-compiled-by-a-more-recent-version-of-the-java-environment )

**Comment by mhoang22** [comment]
Note: in pmbio (https://pmbio.org/module-01-setup/0001/04/01/Software_Installation/), these packages are installed in ~/workspace/bin. for cshl_seqtec2023, i installed these packages in ~/bin.
And the **Pychopper** installation was unsuccessful so we will **need a docker image** for that.
